drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/utils/ScenarioSimulationGridHeaderUtilities.java

```diff
@@ -136,11 +155,4 @@ public static String getPropertyName(final Simulation simulation, final int colu
                 .map(ExpressionElement::getStep)
                 .collect(Collectors.toList()));
     }
-
-    public static boolean isHeaderEditable(BaseGridRendererHelper rendererHelper, ScenarioHeaderMetaData clickedScenarioHeaderMetadata, ScenarioGridColumn scenarioGridColumn) {
```

Comment:
@jomarko 
I agree that maybe this two methods are redundant and could be merged, but I can not see where the _if/check_ that there was in the deleted one have gone

Reply:
@gitgabrio The checks are moved into `ScenarioSimulationGridHeaderUtilities.isEditableHeader`. I omitted just the `rendererHelper == null` check.